docs/docs/self-host/env.mdx

```diff
@@ -130,15 +130,20 @@ Store uploads in a Google Cloud Storage bucket.
 Some features in self-hosted GrowthBook are only available with a commercial license key. You can get a trial
 Enterprise key from your self-hosted version of GrowthBook, or reach out to [sales@growthbook.io](mailto:sales@growthbook.io) to learn more.
 
-When you have a license key, it should be set to the environment variable `LICENSE_KEY`. The key will be a long string, and should be set to the variable without any spaces or newlines.
+### License Keys
 
-## Enterprise SSO
+If you have a license key, you can activate it in GrowthBook two different ways:
 
-To enable SSO on your self hosted GrowthBook instance, you will need a license key added to your environment variables, and then add
+1. Navigate to **Settings** > **General** and look for the License section of the page. There you can input or edit your license key string. (Recommended)
+2. Alternately, set the environment variable `LICENSE_KEY` to your license key string. If using Docker, go to _docker-compose.yml_ and add the variable to the _growthbook_ > _environment_ section.
+
+### Enterprise SSO
+
+To enable SSO on your self hosted GrowthBook instance, you will need an active license key (see above section), and then you may add
 the SSO settings for your provider. If your provider is not listed below, you can use the generic Open ID Connect below. The JSON
 object with the settings will need to be JSON encoded and then set to the environment variable `SSO_CONFIG`.
 
-### Generic Open ID Connect
+#### Generic Open ID Connect
```

Comment:
The reason I didn't go 4 deep is that they no longer show up on the right navigation.